framework/helpers/BaseStringHelper.php

```diff
@@ -338,4 +338,35 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * Masks a token to make it uncompressible.
```

Comment:
Have updated the description, chose to still include the fact that it makes the string uncompressible, which is the method it uses to mitigate BREACH attacks.